.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+## Description
+
+Replace this paragraph with a description of what this PR is doing. Include
```

Comment:
Maybe format this differently with italic or surround with brackets. Since this section is supposed to be replaced an nothing else below is